packages/app/cypress/e2e/reporter_header.cy.ts

```diff
@@ -38,26 +38,62 @@ describe('Reporter Header', () => {
-      cy.get('.testing-preferences-toggle').should('not.have.class', 'open').click()
-      cy.get('.testing-preferences-toggle').should('have.class', 'open')
-      cy.get('.testing-preferences').should('be.visible')
-      cy.get('.testing-preferences-toggle').click()
-      cy.get('.testing-preferences-toggle').should('not.have.class', 'open')
-      cy.get('.testing-preferences').should('not.exist')
+    const switchSelector = '[data-cy=auto-scroll-switch]'
+
+    context('preferences menu', () => {
+      beforeEach(() => {
+        cy.scaffoldProject('cypress-in-cypress')
+        cy.openProject('cypress-in-cypress')
+        cy.startAppServer()
+        cy.visitApp()
+        cy.contains('dom-content.spec').click()
+        cy.waitForSpecToFinish()
+      })
+
+      it('clicking the down arrow will open a panel showing Testing Preferences', () => {
+        cy.get('.testing-preferences-toggle').trigger('mouseover')
+        cy.get('.cy-tooltip').should('have.text', 'Open Testing Preferences')
+
+        cy.get('.testing-preferences').should('not.exist')
+        cy.get('.testing-preferences-toggle').should('not.have.class', 'open').click()
+        cy.get('.testing-preferences-toggle').should('have.class', 'open')
+        cy.get('.testing-preferences').should('be.visible')
+        cy.get('.testing-preferences-toggle').click()
+        cy.get('.testing-preferences-toggle').should('not.have.class', 'open')
+        cy.get('.testing-preferences').should('not.exist')
+      })
+
+      it('will show a toggle beside the auto-scrolling option', () => {
+        cy.get('.testing-preferences-toggle').click()
+        cy.get(switchSelector).invoke('attr', 'aria-checked').should('eq', 'true')
+        cy.get(switchSelector).click()
+        cy.get(switchSelector).invoke('attr', 'aria-checked').should('eq', 'false')
+      })
     })
 
-    it('will show a toggle beside the auto-scrolling option', () => {
-      const switchSelector = '[data-cy=auto-scroll-switch]'
+    it('does NOT toggle off the user preferences auto-scroll if auto-scroll is temporarily disabled', () => {
+      cy.scaffoldProject('cypress-in-cypress')
+      cy.openProject('cypress-in-cypress')
+      cy.startAppServer()
+      cy.visitApp()
+      cy.contains('dom-content-scrollable-commands.spec').click()
+
+      // wait for the test to scroll all the way to the bottom
+      cy.get(':contains("checks for list items to exist - iteration #25") + :contains("passed")', {
+        timeout: 20000,
+      }).should('exist')
+
+      // then, use the runnable container to fire the scroll events that previously would override and sync the preference config
+      cy.get('[data-cy="reporter-panel"] .reporter > header + .container').its('0').then(($runnableContainer) => {
+        // scroll the container to the top.
+        // fire multiple scroll events so our scroller component believes the scroll came from an actual user.
+        [...Array(10).keys()].forEach(() => {
```

Comment:
Wonder if it's work a comment linking to https://github.com/cypress-io/cypress/blob/develop/packages/reporter/src/lib/scroller.ts#L37 to explain why this works?

Reply:
I would think so. My only concern is if the file changed and the reference became out of data